raiden_installer/constants.py

```diff
@@ -0,0 +1,31 @@
+import sys
+from pathlib import Path
+from xdg import XDG_DATA_HOME, XDG_CONFIG_HOME
+
+
+'''
+Raiden installer uses the XDG Base Directory standard for
+storing its files. XDG_DATA_HOME is used for storing user
+keystore files whilst XDG_CONFIG_HOME stores config files
+necessary for initializing Raiden.
+'''
+BINARY_DIR = Path('/usr/local/bin')
+CONFIG_DIR = Path(XDG_CONFIG_HOME).joinpath('Raiden')
+CONFIG_DIR.mkdir(parents=True, exist_ok=True)
+KEYSTORE_DIR = Path(XDG_DATA_HOME).joinpath('Raiden')
+KEYSTORE_DIR.mkdir(parents=True, exist_ok=True)
+
+
+PLATFORM = 'macOS' if sys.platform == 'darwin' else 'linux'
```

Comment:
The PLATFORM variable stores either 'macOS' or 'linux' depending on what sys.platform returns. I need this string to building the Raiden download URL for the correct system. How would I do it otherwise?

We are not supporting Windows? Or am I misunderstanding your comment?

Reply:
The issue I am pointing out here is that you are defining a variable that is (a) being used in only one place (the function that builds the URL), (b) whose name (`PLATFORM`) indicates something that already something that python interpreter can give you and (c) slightly differs from the standard only for the sake of the function that you are implementing.

All in all, the overall point is that this does not need to be a "constant" in the program. It will be better to remove this definition altogether and just have the function that builds to URL to contain its own logic about how to build the URL.

Reply:
Re: "not supporting windows", you are right that we are not supporting it at the moment.

However, it is always good to be wary of possible changes and the impact on the code you are creating. By having your code rely on the condition that "it is never going to run on windows", you are making the application very fragile to changes.